sdk/contentsafety/Azure.AI.ContentSafety/CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 # Release History
 
-## 1.0.0-beta.1 (2023-05-22)
+## 1.0.0-beta.1 (2023-06-06)
```

Comment:
Did you just not hit this date? In general, we leave this as "Unreleased" until right before we ship.

Reply:
Yes we missed the 0522 release date, and decided to release by 06/06. Sure I will keep "Unreleased" in following versions. Since the date is very near now, can we leave it "2023-06-06" for now?